packages/cube/manifest.ttl

```diff
@@ -0,0 +1,54 @@
+@base <https://barnard59.zazuko.com/> .
+@prefix b59: <https://barnard59.zazuko.com/vocab#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix code: <https://code.described.at/> .
+@prefix p: <https://pipeline.described.at/> .
+
+# renamed from <https://barnard59.zazuko.com/operations/rdf/cube.js#buildCubeShape>
+<operations/cube/buildCubeShape> a p:Operation, p:WritableObjectMode, p:ReadableObjectMode;
+  rdfs:label "build Cube Shape";
+  rdfs:comment "Builds a Cube Shape based on Cube Observation datasets";
+  code:implementedBy [ a code:EcmaScriptModule;
+    code:link <node:barnard59-cube/cube.js#buildCubeShape>
+  ].
+
+# renamed from <https://barnard59.zazuko.com/operations/rdf/cube.js#toObservation>
+<operations/cube/toObservation> a p:Operation, p:WritableObjectMode, p:ReadableObjectMode;
+  rdfs:label "to Cube Observation";
+  rdfs:comment "Converts a set of quads to a Cube Observation";
+  code:implementedBy [ a code:EcmaScriptModule;
+    code:link <node:barnard59-cube/cube.js#toObservation>
+  ].
+
+
+<command/cube/fetch-constraint>
+  a b59:CliCommand ;
+  b59:command "fetch-constraint" ;
+  rdfs:label "Retrieves cube constraint from SPARQL endpoint" ;
+  b59:source "barnard59-cube/pipeline/cube-validation.ttl" ;
+  b59:pipeline <http://barnard59.zazuko.com/pipeline/cube-validation/fetch-cube-constraint> 
+.
+
+<command/cube/check-constraint>
+  a b59:CliCommand ;
+  b59:command "check-constraint" ;
+  rdfs:label "Validate input constraint against profile shapes" ;
+  b59:source "barnard59-cube/pipeline/cube-validation.ttl" ;
+  b59:pipeline <http://barnard59.zazuko.com/pipeline/cube-validation/check-cube-constraint> 
+.
+
+<command/cube/fetch-observations>
+  a b59:CliCommand ;
+  b59:command "fetch-observations" ;
+  rdfs:label "Retrieves cube observations from SPARQL endpoint" ;
+  b59:source "barnard59-cube/pipeline/cube-validation.ttl" ;
+  b59:pipeline <http://barnard59.zazuko.com/pipeline/cube-validation/fetch-cube-observations> 
+.
+
+<command/cube/check-observations>
+  a b59:CliCommand ;
+  b59:command "check-observations" ;
+  rdfs:label "Validate input observations against cube constraint" ;
+  b59:source "barnard59-cube/pipeline/cube-validation.ttl" ;
```

Comment:
I'd place each pipeline in its own file. Are they all in `cube-validation.ttl` to avoid duplication?

Reply:
yes but the actual duplication would be very little